components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -252,7 +252,9 @@ class ContributionFlowSuccess extends React.Component {
                     />
                   </P>
                 </Box>
-                <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={!isEmbed} />
+                <Link href={{ pathname: order.toAccount.name }}>
```

Comment:
@ersachin3112 : This will not work since the `orcer.toAccount.name` is the name not the slug; try using,

```suggestion
                <Link href={{ pathname: order.toAccount.slug }}>
```

Reply:
done 👍🏻 @SudharakaP